src/utils/puns.ts

```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
   'Swim back to School of FSH',
   'You need to mullet over',
+  "You're krilling me!",
+  "You're fluking it up!",
```

Comment:
Might this offend some of our more sensitive users?

Reply:
I thought it was kind of edgy... I'll cut it out.